[Workers] wrangler v3 updated local testing docs (#8691)
cloudflare/cloudflare-docs: content/workers/learning/debugging-workers.md

[2020-05-28 10:42:36] GET example.com/sw.js HTTP/1.1 200 OK
```

In the output above, you can begin to see log lines for the URLs being requested locally. 

{{<Aside type="info">}}

Unlike `wrangler dev`, which provides local instances of resources to develop against, `wrangler dev --remote` will leverage remote resources specified in your `wrangler.toml`.

{{</Aside>}}

> ### Additional info

Reviewer: ```suggestion
### Customize `wrangler dev`
```

You can customize how `wrangler dev` works to fit your needs. Refer to [the `wrangler dev` documentation](/workers/wrangler/commands/#dev) for available configuration options.

{{<Aside type="warning">}}

There is a bug associated with `wrangler dev --remote` documented in the [Known issues section](/workers/platform/known-issues/#wrangler-dev).

{{</Aside>}}

## Getting logs from deployed Workers

> If you need more insight into currently deployed Workers, you can start a new session to livestream logs by running `wrangler tail`.

Reviewer: ```suggestion
To get more insight into currently deployed Workers, run [`wrangler tail`](/workers/wrangler/commands/#tail). `wrangler tail` starts a session to livestream logs from a deployed Worker.
```
Matching Commands page verbiage.
